functorch/functorch/csrc/BatchRulesFactory.cpp

```diff
@@ -9,6 +9,26 @@
 
 namespace at { namespace functorch {
 
+template <typename A, A a, typename C>
+struct NewBlahBatchRuleHelperSymInt;
+
+template <typename F, F Func, typename A, typename B, typename... T>
+struct NewBlahBatchRuleHelperSymInt<F, Func, typelist<A, B, T...>> {
+  static std::tuple<Tensor,optional<int64_t>> apply(
+      const Tensor& tensor,
+      optional<int64_t> batch_dim,
+      SymIntArrayRef shape,
+      T... extra_args) {
+    const auto bdim_size = tensor.sym_size(batch_dim.value());
+    // TODO: optimize this
+    std::vector<c10::SymInt> new_shape;
```

Comment:
I guess I'm supposed to use SmallVector; for some reason I swear this didn't work